test/integration/architecture_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class ArchitectureTest < ActionDispatch::IntegrationTest
+class ArchitectureIntegrationTest < ActionDispatch::IntegrationTest
```

Comment:
`parallel_tests` loads all classes at some point (before splitting the load I guess), so this name would clash with the unit test ArchitectureTest